rpkg_src/decrypt_packagedefinition_thumbs.cpp

```diff
@@ -21,16 +22,21 @@ void generic_function::decrypt_packagedefinition_thumbs(std::string& input_path,
 
     uint64_t packagedefinitions_thumbs_file_size = file.tellg();
 
-    int packagedefinitions_thumbs_header_size = 20;
+    int packagedefinitions_thumbs_header_size = 16;
 
-    packagedefinitions_thumbs_file_size -= packagedefinitions_thumbs_header_size;
+	packagedefinitions_thumbs_file_size -= packagedefinitions_thumbs_header_size;
+	packagedefinitions_thumbs_file_size -= 4; // -4 to skip the checksum
```

Comment:
```suggestion
    packagedefinitions_thumbs_file_size -= packagedefinitions_thumbs_header_size;
    packagedefinitions_thumbs_file_size -= 4; // -4 to skip the checksum
```

Reply:
Blame Visual Studio. Fixed in another commit.